packages/terra-signature/src/terra-dev-site/doc/example/SignatureExample.jsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react';
+import Select from 'terra-form-select';
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
+import Signature from 'terra-signature/lib/Signature';
+
+class SignatureExample extends React.Component {
+  constructor() {
+    super();
+
+    this.state = { lineSegments: [], lineWidth: Signature.Opts.Width.FINE };
+
+    this.handleSingleLine = this.handleSingleLine.bind(this);
+    this.handleClear = this.handleClear.bind(this);
+    this.handleLineWidth = this.handleLineWidth.bind(this);
+  }
+
+  handleSingleLine() {
+    const singleLine = [{ x1: 71, y1: 8, x2: 71, y2: 8 }, { x1: 71, y1: 8, x2: 71, y2: 10 }, { x1: 71, y1: 10, x2: 71, y2: 17 }, { x1: 71, y1: 17, x2: 71, y2: 28 }, { x1: 71, y1: 28, x2: 71, y2: 44 }, { x1: 71, y1: 44, x2: 71, y2: 56 }, { x1: 71, y1: 56, x2: 71, y2: 68 }, { x1: 71, y1: 68, x2: 71, y2: 75 }, { x1: 71, y1: 75, x2: 71, y2: 81 }, { x1: 71, y1: 81, x2: 71, y2: 84 }, { x1: 71, y1: 84, x2: 71, y2: 86 }, { x1: 71, y1: 86, x2: 71, y2: 87 }, { x1: 71, y1: 87, x2: 71, y2: 87 }];
+    const newState = Object.assign({}, this.state, { lineSegments: singleLine });
+    this.setState(newState);
+  }
+
+  handleClear() {
+    const newState = Object.assign({}, this.state, { lineSegments: [] });
+    this.setState(newState);
+  }
+
+  handleLineWidth(e) {
+    const newState = Object.assign({}, this.state, { lineSegments: [], lineWidth: parseInt(e.target.value, 10) });
+    this.setState(newState);
+  }
+
+  render() {
+    return (
+      <div style={{ height: '100px', width: '100%', border: '1px solid black' }} >
+        <Signature id="foo" lineWidth={this.state.lineWidth} lineSegments={this.state.lineSegments} />
+        <button onClick={this.handleClear}>Clear </button>
+        <button onClick={this.handleSingleLine}>Sign w/Line </button>
+        <Select
```

Comment:
Need to update this to use the new select component/API

Reply:
Resolved in this commit: https://github.com/cerner/terra-core/pull/1542/commits/e991c4c771f6ece77c547539d3bcd7dcb7cbd400